<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="OB-Sum" failures="1" tests="3" name="TC_36_3_ExpenseHead_7001_7035__JournalEntry_Bank_Supplier" time="207.403" errors="0" timestamp="2021-12-30T13:02:10 IST">
  <testcase classname="Test_Trial_Balance_RE_Jour_Bank_Supp.TC_36_3_ExpenseHead_7001_7035__JournalEntry_Bank_Supplier" name="GoToUrlAssertUrlPositiveEnternLOGINPositiveEnternPASSClickLoginGoToUrlAssertUrlClickAgentsP1" time="48.089">
    <failure type="java.lang.AssertionError" message="Unable to locate object: By.xpath: //A[@id=&amp;apos;ctl00_cPH_btnAddJournalEntry&amp;apos;]">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //A[@id='ctl00_cPH_btnAddJournalEntry']
at org.testng.Assert.fail(Assert.java:99)
at Trial_Balance.JournalEntry.Click_plusiconclick(JournalEntry.java:154)
at Test_Trial_Balance_RE_Jour_Bank_Supp.TC_36_3_ExpenseHead_7001_7035__JournalEntry_Bank_Supplier.GoToUrlAssertUrlPositiveEnternLOGINPositiveEnternPASSClickLoginGoToUrlAssertUrlClickAgentsP1(TC_36_3_ExpenseHead_7001_7035__JournalEntry_Bank_Supplier.java:105)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlPositiveEnternLOGINPositiveEnternPASSClickLoginGoToUrlAssertUrlClickAgentsP1 -->
  <testcase classname="Test_Trial_Balance_RE_Jour_Bank_Supp.TC_36_3_ExpenseHead_7001_7035__JournalEntry_Bank_Supplier" name="GoToUrlAssertUrlPositiveEnternLOGINPositiveEnternPASSClickLoginGoToUrlAssertUrlClickAgentsP2" time="75.878"/>
  <testcase classname="Test_Trial_Balance_RE_Jour_Bank_Supp.TC_36_3_ExpenseHead_7001_7035__JournalEntry_Bank_Supplier" name="GoToUrlAssertUrlPositiveEnternLOGINPositiveEnternPASSClickLoginGoToUrlAssertUrlClickAgentsP3" time="77.039"/>
</testsuite> <!-- TC_36_3_ExpenseHead_7001_7035__JournalEntry_Bank_Supplier -->
